README.md

```diff
@@ -34,13 +41,31 @@ Komar can also be used as a separate unit. In this case, a custom phase connecto
 
 ## Construction and usage
 
-Komar itself consists of a power stage PCB with Mitochondrik and the housing (aluminum enclosure plus several plastic parts). The PCB may use several optional 3D-printed parts that don't influence its properties directly, but may improve the overall user experience:
+Komar itself consists of a power stage PCB with Mitochondrik and the housing (aluminum enclosure plus several plastic parts).
+The PCB may use several optional 3D-printed parts that don't influence its properties directly, but may improve the overall user experience:
 
 - Lightpipe
 - Capacitor support
 
 Please refer to the manufacturing manual for further information.
 
+## Thermal performance
+
+Thermal simulation is performed under following conditions:
+
+* The heat source is applied to the heatsink area where the PCB is placed. Its power is 78.5 W.
+It is a typical value of power losses in the Komar power stage when operating at a power level of 2500 W
+and powered from a discharged 12S battery (~40 V). This should represent the worst-case scenario.
+
+* Ambient temperature is 20 °C
+
+* Convection value of 350 is defined empirically after conducting several experiments with actual hardware.
+It may be considered a moderate convection estimation as during the laboratory testing the device mounting point was suboptimal.
```

Comment:
```suggestion
```

Here we should either specify what exactly makes the estimate pessimistic or we should say nothing.